docs/code-style.md

```diff
@@ -0,0 +1,10 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 Arthur Breitman
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+   -->
+
+Our style guides can be found in https://github.com/serokell/style.
+This file can be a copy-paste of existing style guide (possibly with some
+modifications) or contain a link (or links) to style guides (and maybe
+describe some changes).
```

Comment:
This needs to be replaced, I suggest pointing at morley's style guide.
```suggestion
Haskell code adheres to Morley's style-guide, which can be found here:
https://gitlab.com/morley-framework/morley/-/blob/master/docs/code-style.md

There is no particular style guide for LIGO code.
```

Reply:
Updated.